SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -265,7 +265,8 @@
     ),
     TrackingAction = cms.PSet(
         DetailedTiming = cms.untracked.bool(False),
-        CheckTrack = cms.untracked.bool(False)
+        CheckTrack = cms.untracked.bool(False),
+        DoFineCalo = cms.untracked.bool(True)
```

Comment:
Should this be enabled by default (or is it just for PR testing purposes)?

It might be useful to introduce a ProcessModifier (https://github.com/cms-sw/cmssw/tree/master/Configuration/ProcessModifiers/python) to make it easier to activate on demand

Reply:
Just for PR testing purposes now. It is indeed annoying to have to set the flag in 3 different places, so I'll learn about this ProcessModifier.